packages/muelu/src/Transfers/Classical/MueLu_ClassicalMapFactory_def.hpp

```diff
@@ -82,11 +82,9 @@
 
 // NOTE: We should be checking for KokkosKernels here, but
 // MueLu doesn't have a macro for that
-#ifdef HAVE_MUELU_KOKKOSCORE
 #include "MueLu_LWGraph_kokkos.hpp"
 #include <KokkosGraph_Distance1ColorHandle.hpp>
 #include <KokkosGraph_Distance1Color.hpp>
-#endif
```

Comment:
The two "KokkosGraph" headers are coming from Kokkos Kernels which is not a hard dependency, they should be guarded with `#ifdef HAVE_MUELU_KOKKOSKERNELS`.

Reply:
addressed